pinot-common/src/test/java/org/apache/pinot/sql/parsers/TableNameExtractorTest.java

```diff
@@ -16,11 +16,12 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.client;
+package org.apache.pinot.sql.parsers;
```

Comment:
There's still a mismatch - the main class is in `org.apache.pinot.sql.parsers.parser`